### DISCUSSION TITLE
Should the British/American spellings be unified?

### DISCUSSION THREAD

**Discussion by phuclv90** [ImageMagick/ImageMagick#2848] [Development]
The repo contains both [American ](https://github.com/ImageMagick/ImageMagick/search?q=color&type=code) and [British](https://github.com/ImageMagick/ImageMagick/search?q=colour) spellings:

- `"Magick++/Color.h"`
- `unsigned char colour_palette[]`
- `DDSSingleColourLookup`
- ...

I think the spelling should be consistent

**Comment by snibgo** [comment]
Well, some people know how to spell "colour", and some people don't.

A quick grep of the *.c source code finds the American "color" 15362 times, and the British "colour" a mere 95 times. I suppose I am out-voted.

**Comment by dlemstra** [comment]
This is probably not the only word that is written differently but I just replaced all occurrences of `colour` with `color`.
